google-oauth-client/src/main/java/com/google/api/client/auth/openidconnect/IdTokenVerifier.java

```diff
@@ -142,9 +204,87 @@ public final String getIssuer() {
    * @return {@code true} if verified successfully or {@code false} if failed
    */
   public boolean verify(IdToken idToken) {
-    return (issuers == null || idToken.verifyIssuer(issuers))
-        && (audience == null || idToken.verifyAudience(audience))
-        && idToken.verifyTime(clock.currentTimeMillis(), acceptableTimeSkewSeconds);
+    boolean simpleChecks =
```

Comment:
`simpleChecks` and the subsequent `if (!simpleChecks)` are difficult to read for me. Do you have a better variable name? `IssuerAudienceAndTokenTimeAreValid` might be too long.

Reply:
renamed to tokenFieldsValid

Reply:
That's clear. Thanks.